src/coreclr/jit/loopcloning.cpp

```diff
@@ -392,36 +393,40 @@ JitExpandArrayStack<JitExpandArrayStack<LC_Condition>*>* LoopCloneContext::Ensur
 #ifdef DEBUG
 void LoopCloneContext::PrintBlockConditions(unsigned loopNum)
 {
+    printf("Block conditions:\n");
+
     JitExpandArrayStack<JitExpandArrayStack<LC_Condition>*>* levelCond = blockConditions[loopNum];
     if (levelCond == nullptr || levelCond->Size() == 0)
     {
-        JITDUMP("No block conditions\n");
+        printf("No block conditions\n");
```

Comment:
Any reason why these and below are made `printf` instead of `JITDUMP`?

Reply:
JITDUMP checks verbose (is COMPlus_JitDump set); printf does not. This means a function like this can be called from the debugger even if verbose isn't set. Of course, the caller probably wants to check `verbose`. In general, if `verbose` is already checked, I prefer direct `printf`

Reply:
Ah, didn't realize that this method is called under `verbose`.